torchvision/prototype/transforms/_augment.py

```diff
@@ -84,6 +88,8 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
             break
         else:
             i, j, h, w, v = 0, 0, img_h, img_w, image
+            # FYI: Now taht we are not JIT-scriptable, I probably can avoid copying-pasting the image to itself in this
+            # scenario. Perhaps a simple clone would do.
```

Comment:
Yeah, there is major cleanup coming up. A lot of the annotations on the transforms are either wrong or to strict to appease torchscript. We should revisit before we promote out of prototype. There is also a user request for this: #5398

Reply:
Sounds good. There is a non-trivial number of clean ups we need to do prior landing this API to the main area. Please create an issue that describes these clean ups (see this [example](https://github.com/pytorch/vision/issues/4679)). I would also recommend creating a Project to keep track of all the related tickets (see this [example](https://github.com/pytorch/vision/projects/2)).